front/app/components/HookForm/SelectionChoiceSetup/index.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
This component is giving me a lot of warnings in the console, and it's also not working correctly. If I add an option by typing and pressing 'enter', the whole form revalidates and any options that I entered before disappear

Reply:
Missed setting a "key" for the list items - caused the console errors. Thanks!! :) I'll look into the weird behaviour on the "Enter' key down :+1: